gadopt/approximations.py

```diff
@@ -75,19 +75,32 @@ class BoussinesqApproximation(BaseApproximation):
     Viscous dissipation is neglected (Di << 1)."""
     compressible = False
 
-    def __init__(self, Ra, kappa=1, g=1, rho=1, alpha=1):
+    def __init__(self, Ra, rho=1, g=1, alpha=1, kappa=1, RaB=0, delta_rho=1, H=0):
```

Comment:
Shouldn't affect your uses of `BoussinesqApproximation` or any of the existing demos, but I wonder if maybe this should enforce keyword-only args. Lots of numbers whose order could get confused!

```suggestion
    def __init__(self, Ra, *, rho=1, g=1, alpha=1, kappa=1, RaB=0, delta_rho=1, H=0):
```